mono/mini/mini-runtime.c

```diff
@@ -1905,6 +1905,97 @@ mono_jit_map_is_enabled (void)
 
 #endif
 
+#if ENABLE_JIT_DUMP
+static FILE* perf_dump_file;
+uint64_t codeIndex = 0;
+pthread_mutex_t mutex = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
`mutex` is too broad a name to be the name of a global variable. Please make this static, too (so it's not visible from other `.c` files).

Reply:
Yes, but also the type is wrong. it should be a `mono_mutex_t`